File: CampingNeretva/CampingNeretva.Model/ReviewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model
{
    public class ReviewModel
    {
        public int ReviewId { get; set; }
        public WorkerModel Worker { get; set; }
        public UserModel User { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime DatePosted { get; set; }
    }
}

File: CampingNeretva/UI/campingneretva_mobile/lib/models/review_model.dart
import 'package:campingneretva_mobile/models/user_model.dart';
import 'package:campingneretva_mobile/models/worker_model.dart';

class Review {
  final int id;
  final String comment;
  final int rating;
  final String datePosted;
  final User user;
  final Worker worker;

  Review({
    required this.id,
    required this.comment,
    required this.rating,
    required this.datePosted,
    required this.user,
    required this.worker,
  });

  factory Review.fromJson(Map<String, dynamic> json) {
    return Review(
      id: json['reviewId'],
      comment: json['comment'],
      rating: json['rating'],
      datePosted: json['datePosted'],
      user: User.fromJson(json['user']),
      worker: Worker.fromJson(json['worker']),
    );
  }
}
